build/virt-prerunner/Dockerfile

```diff
@@ -16,9 +16,23 @@ FROM alpine
 
 RUN apk add --no-cache tini curl screen dnsmasq cdrkit iptables
 
-RUN curl -sLo /usr/bin/cloud-hypervisor https://github.com/cloud-hypervisor/cloud-hypervisor/releases/download/v24.0/cloud-hypervisor-static && chmod +x /usr/bin/cloud-hypervisor
-RUN mkdir /var/lib/cloud-hypervisor && curl -sLo /var/lib/cloud-hypervisor/hypervisor-fw https://github.com/cloud-hypervisor/rust-hypervisor-firmware/releases/download/0.4.0/hypervisor-fw
-RUN curl -sLo /usr/bin/ch-remote https://github.com/cloud-hypervisor/cloud-hypervisor/releases/download/v24.0/ch-remote-static && chmod +x /usr/bin/ch-remote
+RUN set -eux; \
+    mkdir /var/lib/cloud-hypervisor; \
+    case "$(uname -m)" in \
```

Comment:
Is it possible to support Cross-platform build to make is easier for development

Reply:
It should be possible with some changes in the skaffold.yaml file, but I think it's not worth it to keep it in the repo